conf/map/battle/feature.conf

```
@@ -98,4 +98,7 @@ features: {
 	// true:  enable
 	// false: disable (default)
 	replace_refine_npcs: false
+
+	// Enable Multi-Critical damage
+	enable_multi_critical: true
```

```suggestion
	// Enable Multi-Critical damage
	// Requires: 2016-12-07 RagexeRE or later
	enable_multi_critical: true
```
perhaps follow the format like other config too.